### ISSUE TITLE
provide arm64 compatible binary

### ISSUE THREAD

**Issue by rr-nick-tan** [liquibase/liquibase-package-manager#131]
**Is your feature request related to a problem? Please describe.**

we run our CI system on AWS Graviton instances (arm64 cpu chipset) for better price performance, the current lpm cli is for x86_64 only.
if run on arm64, i see error: `/liquibase/bin/lpm: cannot execute binary file: Exec format error`

**Describe the solution you'd like**
we can build the binaries for both arm64 and x86_64:
https://github.com/liquibase/liquibase-package-manager/blob/master/Makefile#L18
```
GOOS=linux GOARCH=amd64 go build -o $(PWD)/bin/linux/lpm $(PWD)/cmd/lpm/darwin.go
GOOS=linux GOARCH=arm64 go build -o $(PWD)/bin/linux/lpm $(PWD)/cmd/lpm/darwin.go
```

**Describe alternatives you've considered**

**Additional context**

**Comment by mcred** [comment]
H,i @rr-nick-tan. The release for [v0.1.7](https://github.com/liquibase/liquibase-package-manager/releases/tag/v0.1.7) has been updated to include an ARM release for linux. 

Would you download this release and let me know if you run into any issues? I don't have an ARM linux system available to test this myself.

**Comment by rr-nick-tan** [comment]
@mcred thanks for the quick fix, [lpm-0.1.7-linux-arm64.zip](https://github.com/liquibase/liquibase-package-manager/releases/download/v0.1.7/lpm-0.1.7-linux-arm64.zip) works perfectly

**Comment by mcred** [comment]
Glad to hear it! In future releases we'll probably split out amd and arm releases to be specific. If you're looking to automate this install, keep an eye out.
